### ISSUE TITLE
attr 'maxWidth' already defined - conflict with android's preference support libraries

### ISSUE THREAD

**Issue by ZakTaccardi** [gcacace/android-signaturepad#50]
```
def supportLibraryVersion = '23.2.1'

compile "com.android.support:preference-v7:$supportLibraryVersion"
compile "com.android.support:preference-v14:$supportLibraryVersion"
compile "com.github.gcacace:signature-pad:1.1.0@aar"
```

Error on build: `Error:(187) Attribute "maxWidth" has already been defined`

**Comment by MarKayS** [comment]
I have exactly same problem, did anybody find any solution / workaround to this?

**Comment by gcacace** [comment]
It's due to this: https://code.google.com/p/android/issues/detail?id=22576
I'll investigate how to fix that soon. In the meantime feel free to fork the lib and change the attribute name into something else.

**Comment by smaspe** [comment]
what is usually done is to rename your attribute, e.g. by prefixing with something indicating it belongs to your library. like for example in ViewPagerIndicator: https://github.com/JakeWharton/ViewPagerIndicator/blob/master/library/res/values/vpi__attrs.xml

**Comment by gcacace** [comment]
Yes, that's a breaking change and I will release it soon.

**Comment by dandar3** [comment]
If to be changed I would vote for changing into something more meaningful like _minPenWidth_ and _maxPenWidth_ or something similar.

**Comment by scriptori** [comment]
Any status on this issue? I need to use com.android.support:preference-v7:23.3.0 and this problem is interfering it.

**Comment by gcacace** [comment]
Fixed in v1.2.0.

**Comment by scriptori** [comment]
Thanks!
